commit 663d0922696425b5de2ef055191c76e1b945064b
Author: Josh Wilsdon <jwilsdon@joyent.com>
Date:   2019-04-23T13:52:17-07:00 (6 months ago)
    
    OS-7758 test-vm fails due to out of quota instead of show write
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/src/vm/tests/test-update.js b/src/vm/tests/test-update.js
index 144c610d..dc44578b 100644
--- a/src/vm/tests/test-update.js
+++ b/src/vm/tests/test-update.js
@@ -20,7 +20,7 @@
  *
  * CDDL HEADER END
  *
- * Copyright (c) 2018, Joyent, Inc.
+ * Copyright (c) 2019, Joyent, Inc.
  *
  */
 
@@ -1286,7 +1286,7 @@ test('fill up zoneroot', function (t) {
     execFile('/usr/bin/dd', [
         'if=/dev/zero',
         'of=/zones/' + vm_uuid + '/root/zeros',
-        'bs=1M'
+        'bs=37M' // Use a value that doesn't go evenly into the 1GB VM quota.
     ], function (err, stdout, stderr) {
         var match = stderr.match(/dd: unexpected short write, wrote/);
         t.ok(match, 'expected short write'
